news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap.md

```diff
@@ -0,0 +1,134 @@
+
+They had managed to get back to an even 4-4 after being down three points, but there were still several difficult picks ahead of the two teams. On Chile's turn to pick, [NM4] was chosen: they were dominating most of the map thanks to ::{ flag=CL }:: [Smukur](https://osu.ppy.sh/users/6434318)'s 1-miss run, the best score on the map this week, but Canada recovered almost all their deficit on the map at the very end, ending up in an extremely close 6,566 points difference. Chile had now won four maps in a row in a miraculous turn of events, and it was up to Canada to pick. They finally settled on [DT3](https://osu.ppy.sh/beatmapsets/2191178#fruits/4634726) as the last pick of the match, but Chile prevailed, with an another SS from ::{ flag=CL }:: [Aryssiel](https://osu.ppy.sh/users/2389481), ending the match on a five maps win streak to win 6-4. 
+
+Thanks to that win, Chile secured themselves a spot in the final 6, a result that they didn't achieve since 2020. Canada, on the other hand, will still be competing, and I'm sure are determined to make the most out of that losers' bracket run.
+
+### ::{ flag=JP }:: Japan vs. ::{ flag=GB }:: United Kingdom
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="NEED TO CHANGE LINK" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Japan and the United Kingdom are both teams that we've rarely seen access the later rounds of the tournament in recent years. Japan managed to reach them last year for the first time since 2018 before being eliminated, finishing top 12, while the United Kingdom never made it past the Quarterfinals and the final 12 (at least since the introduction of the losers' bracket in 2015). This year though, these two teams are looking better than they have for a long time, and are looking to beat their previous results. Japan, getting seed 8, looked very promising from the very beginning, with several of their players improving a lot throughout the year. As for the United Kingdom, several of their veteran players started grinding the game again, and their newcomers keep on improving at a rapid pace. Both teams were sent to the losers' bracket in the previous week, but ended up winning their first Quarterfinals match convincingly, 6-1 against Sweden for Japan, and 6-0 against the Netherlands for the United Kingdom. Now meeting in an elimination match, one of these teams would end their CWC run here, despite their desire to keep on going and show what they're capable of.
```

Comment:
```suggestion
Japan and the United Kingdom are both teams that we've rarely seen access the later rounds of the tournament in recent years. Japan managed to reach them last year for the first time since 2018 before being eliminated and finishing at top 12, while the United Kingdom never made it past the Quarterfinals and the final 12 (at least since the introduction of the losers' bracket in 2015).

This year though, these two teams are looking better than they have for a long time, and are looking to beat their previous results. Japan, getting seed 8, looked very promising from the very beginning, with several of their players improving a lot throughout the year. As for the United Kingdom, several of their veteran players started grinding the game again, and their newcomers keep on improving at a rapid pace. Both teams were sent to the losers' bracket in the previous week, but ended up winning their first Quarterfinals match convincingly, 6–1 against Sweden, and 6–0 against the Netherlands. Now meeting in an elimination match, one of these teams would end their CWC run here, despite their desire to keep on going and show what they're capable of.
```